WordPress/Classes/Services/MediaService.h

```diff
@@ -59,7 +59,7 @@ typedef NS_ERROR_ENUM(MediaServiceErrorDomain, MediaServiceError) {
  @failure a block that will be invoked when there is upload error.
  */
 - (void)updateMedia:(nonnull Media *)media
-     fieldsToUpdate:(NSArray<NSString *> *)fieldsToUpdate
+     fieldsToUpdate:(nonnull NSArray<NSString *> *)fieldsToUpdate
```

Comment:
I made the assumption that one would never want to call this method by passing `nil` to this parameter.  An the compiler didn't reveal any inconsistency.